scripts/sandbox.py

```
@@ -57,15 +58,33 @@ def main() -> int:
 def buildinputs(
         dockerfile: pathlib.Path | str,
-        platform: Literal["linux/amd64", "linux/arm64", "linux/s390x", "linux/ppc64le"] = "linux/amd64"
+        platform: Literal["linux/amd64", "linux/arm64", "linux/s390x", "linux/ppc64le"] = "linux/amd64",
+        build_args: dict[str, str] | None = None
 ) -> list[pathlib.Path]:
     if not (ROOT_DIR / "bin/buildinputs").exists():
         subprocess.check_call([MAKE, "bin/buildinputs"], cwd=ROOT_DIR)
+    env = {"TARGETPLATFORM": platform, **os.environ}
+    if build_args:
+        env.update(build_args)
     stdout = subprocess.check_output([ROOT_DIR / "bin/buildinputs", str(dockerfile)],
```

Ruff (0.14.13)

`subprocess` call: check for execution of untrusted input

(S603)